projects/packages/my-jetpack/_inc/components/add-license-screen/index.jsx

```diff
@@ -23,11 +24,26 @@ export default function AddLicenseScreen() {
 		restApi.setApiNonce( apiNonce );
 	}, [] );
 
+	const { recordEvent } = useAnalytics();
+
+	const onClickGoBack = useCallback(
+		event => {
+			recordEvent( 'jetpack_myjetpack_license_activation_back_link_click' );
+
+			if ( -1 !== document.referrer.indexOf( window.location.host ) ) {
```

Comment:
```suggestion
                        if ( document.referrer.includes( window.location.host ) ) {
```

Docs: https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/String/includes